src/renderers/web/renderer.ts

```diff
@@ -593,15 +593,17 @@ export class WebRenderer extends NullRenderer implements EventSource<string> {
 			}
 
 			else if (part instanceof parts.Fade) {
-				dialogueAnimationCollection.add("linear", [new Keyframe(0, new Map([
+				let keyframes = [new Keyframe(0, new Map([
```

Comment:
Also omit the first keyframe if `part.start` is 0. It technically doesn't matter for `start` since the second keyframe overriding the first ends up with the correct result as well, but I still prefer it to be consistent.